hazelcast/src/main/java/com/hazelcast/client/impl/ClientEngineImpl.java

```diff
@@ -149,20 +153,36 @@ private ClientExceptionFactory initClientExceptionFactory() {
         return new ClientExceptionFactory(jcacheAvailable);
     }
 
-    private Executor newExecutor() {
+    private Executor newClientExecutor() {
         final ExecutionService executionService = nodeEngine.getExecutionService();
         int coreSize = Runtime.getRuntime().availableProcessors();
 
         int threadCount = node.getProperties().getInteger(GroupProperty.CLIENT_ENGINE_THREAD_COUNT);
         if (threadCount <= 0) {
             threadCount = coreSize * THREADS_PER_CORE;
         }
+        logger.info("Creating new client executor with threadCount=" + threadCount);
```

Comment:
I prefer to be less noisy by default. We print thread count just for operational threads.